pmg/views.py

```diff
@@ -244,7 +247,40 @@ def committees():
 
     logger.debug("committees page called")
     committees = load_from_api('committee', return_everything=True)['results']
-    return render_template('committee_list.html', committees=committees, )
+
+    nat = {
+        'name': 'National Assembly',
+        'committees': []
+    }
+    ncp = {
+        'name': 'National Council of Provinces',
+        'committees': []
+    }
+    jnt = {
+        'name': 'Joint Committees',
+        'committees': []
+    }
+
+    adhoc_committees = OrderedDict((('nat',nat),('ncp',ncp),('jnt',jnt)))
+    reg_committees = deepcopy(adhoc_committees)
+    committees_type = None
+
+    for committee in committees:
+        if committee['ad_hoc'] is True:
+            committees_type = adhoc_committees
+        else:
+            committees_type = reg_committees
+
+        if committee['house_id'] is 3:
```

Comment:
I'd suggest breaking these magic numbers out to constants like Committee.NAT_ASSEMBLY